### DISCUSSION TITLE
Flutter app event for hybrid application

### DISCUSSION THREAD

**Discussion by missterpan** [oddbit/flutter_facebook_app_events#185] [General]
Hi Oddbit, 

How can we track Facebook app events in this circumstance if my app was created with flutter and some parts contain web views? While we can install SDK in a flutter, how can we capture events that occur in the web view area and transform them into app events? We looked at a Facebook document on hybrid mobile app solutions (https://developers.facebook.com/docs/app-events/hybrid-app-events/). Our app is based on flutter but it wasn't fully native as some of the applications including the web app version

Otherwise, we're still confused as to what we should do with the flutter version of the hybrid app event, thus we'd need to know how we may include your app event plugin into our flutter-based hybrid app.

**Comment by SlickSlime** [comment]
If webview's code can be controlled by you then I suppose you can use Facebook Pixel for you web app and send events from there.
